Tools/GenerateWebsiteData.ps1

```diff
@@ -147,9 +201,21 @@ function AnnotateParametersAndSyntaxObjects($cmdletName, $docObj) {
             # the parameter set are honored. It is possible to have one parameter set be a proper
             # subset of another.
             ForEach ($paramSetParam in $parameterSet.Parameters) {
+                # If the parameter is a dynamic parameter, then it won't show up in Get-Help -Full.
+                # So we will have to generate an object that corresponds to this dynamic parameter and add it to $syntaxObj.parameter.
+                if ($paramSetParam.IsDynamic) {
+                    $dynamicParameterSyntaxObject = New-DynamicParamObject $paramSetParam
+
```

Comment:
For readability, I'd remove line 208 and 210. The comment on line 204 does a good job explaining things, and a 4-line if statement body is pretty legible IMHO.